bedrock/newsletter/templates/newsletter/includes/form.html

```diff
@@ -72,7 +72,7 @@
 
     <div class="form-submit">
       <input type="submit" id="footer_email_submit"
-             value="{% if submit_text %}{{ submit_text }}{% else %}{{ _('Sign me up') }} »{% endif %}" class="button">
+             value="{% if submit_text %}{{ submit_text }}{% else %}{{ _('Sign Up Now') }}{% endif %}" class="button-newsletter {{ button_class }}">
```

Comment:
This one affects l10n, and you need to put it behind a en-* conditional like the `Keep up with<br> all things Firefox.Keep up with<br> all things Firefox.`string, since we can't have tags. Once we have coverage, we need to clean up the conditionals.

I suggest to add the code to clean up to this bug 
https://bugzilla.mozilla.org/show_bug.cgi?id=1258998